// magic-script-components/src/components.js
import React from '../../react/index.js';

function Button (props) {
    // return (<button {...props} />);
    return React.createElement('button', props);
}

function Model (props) {
    // return (<model {...props} />);
    return React.createElement('model', props);
}

function Text (props) {
    // return (<text {...props} />);
    return React.createElement('text', props);
}

function View (props) {
    // return (<view {...props} />);
    return React.createElement('view', props);
}

function Toggle (props) {
    return React.createElement('toggle', props);
}

function ScrollBar (props) {
    // return (<scrollBar {...props} />);
    return React.createElement('scrollBar', props);
}

function ScrollView (props) {
    // return (<scrollView {...props} />);
    return React.createElement('scrollView', props);
}

function GridLayout (props) {
    // return (<gridLayout {...props} />);
    return React.createElement('gridLayout', props);
}

function LinearLayout (props) {
    // return (<linearLayout {...props} />);
    return React.createElement('linearLayout', props);
}

function Audio (props) {
    // return (<audio {...props} />);
    return React.createElement('audio', props);
}

export { Audio, Button, GridLayout, LinearLayout, Model, ScrollBar, ScrollView, Text, Toggle, View };
